tests/test_models.py

```
@@ -0,0 +1,28 @@
+# Copyright (C) 2026 Virta Laboratories, Inc.  All rights reserved.
+
+"""Tests for bf_opencore models (mirrors bf_opencore/models)."""
+
+import pytest
+
+
+def test_asset_model(db):
```

Ruff (0.15.1)

[warning] Unused function argument: `db`

(ARG001)